.travis.yml

```diff
@@ -2,7 +2,6 @@ sudo: false
 language: node_js
 node_js:
   - '8'
-  - '6'
```

Comment:
I opened a follow-up PR #61 where I propose to enforce Node 8. Node 6 is quite unstable with `Jest` and `toThrowErrorMatchingSnapshot` matcher. I didn't explore it further as we are already promoting Node 8 and up in Gutenberg. I think we should align those 2 repositories anyway.

Reply:
I will remove ac1b8dc321985c4cdbefe257667ef884952a1387 commit if #61 lands.